src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/AnonymousAccessRequestTest.kt

```diff
@@ -0,0 +1,62 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.resource
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.IntegrationTestBase
+
+/**
+ * Integration tests that make HTTP requests against the API for resources that are designed to be accessed anonymously
+ * without a bearer token.
+ */
+class AnonymousAccessRequestTest : IntegrationTestBase() {
```

Comment:
This looks like a new test class, but is actually a rename of `GetSwaggerSpecTest` with a few extra non swagger related tests (hence the rename)